modules/vpc/outputs.tf

```diff
@@ -68,3 +68,16 @@ output "public_gateway_ids" {
   description = "The IDs of the Public Gateways"
   value       = [for pgw in ibm_is_public_gateway.pgws : pgw.id]
 }
+
+output "subnets" {
+  description = "List of subnets associated with this VPC"
```

Comment:
List of subnets associated with this VPC -> you meant `the created VPC` 
So it can be - List of subnets associated with the VPC